src/main/java/DateTime.java

```diff
@@ -0,0 +1,32 @@
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class DateTime {
+
+    protected String input;
+    protected Date date;
+
+    public DateTime (String input){
+        this.input = input;
+    }
+
+    public void setDateAndTime() throws DukeException {
+        SimpleDateFormat formatter = new SimpleDateFormat("dd/MM/yyyy HHmm");
+
+        try {
+            date = formatter.parse(input);
+        } catch (ParseException e) {
+            throw new DukeException("OOPS!!! Invalid time and date format. Please enter in dd/MM/yyyy HHmm format");
```

Comment:
the difference in caps "dd/MM/yyyy HHmm" might actually be a little messy. Can consider other ways to display it, or just simply use all caps/no caps at all.